modules/persistent-storage-csi-manila-dynamic-provisioning.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-manila.adoc
+
+[id="persistent-storage-csi-manila-dynamic-provisioning_{context}"]
+= Dynamically provisioning Manila CSI volumes
+
+{product-title} installs a StorageClass for each available Manila share type.
+
+The YAML files that are created are completely decoupled from Manila and from its Container Storage Interface (CSI) plug-in. As an application developer, you can dynamically provision ReadWriteMany (RWX) storage and deploy Pods with applications that safely consume the storage using YAML manifests.
+
```

Comment:
Perhaps add "You can also provision other access modes such as ReadWriteOnce (RWO)."   We cover that at line 38, and we want to emphasize RWX here since lots of other drivers don't have support for that mode, but we don't want to give the impression that somehow this driver can *only* do RWX.

Reply:
Yeah, I wondered whether it might lead to that impression. Will include your addition.